docs/examples/mht_example.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+========================================================
+General Multi Hypotheses tracking implementation example
+========================================================
+"""
+
+# %%
+# The multi hypotheses tracking (MHT) algorithm is considered one of the best tracking algorithms,
+# consisting of creating a tree of potential tracks for
+# each target candidate (in a multi-target scenario) and pruning such hypotheses
+# in the data association phase. It is particularly efficient in maintaining trajectories of
+# multiple objects and handling uncertainties and ambiguities of tracks (e.g. presence of
+# clutter).
+#
+# MHT, by definition, has several algorithms that fall under this definition, which
+# include Global Nearest Neighbour (GNN, :doc:`check here <../tutorials/06_DataAssociation-MultiTargetTutorial>`),
+# Joint Probabilistic Data association (JPDA, :doc:`tutorial here <../tutorials/08_JPDATutorial>`),
```

Comment:
I think prefixing the tutorials directory with `auto_` should work

```suggestion
# include Global Nearest Neighbour (GNN, :doc:`check here <../auto_tutorials/06_DataAssociation-MultiTargetTutorial>`),
# Joint Probabilistic Data association (JPDA, :doc:`tutorial here <../auto_tutorials/08_JPDATutorial>`),
```

Reply:
Thanks! This looks like a good suggestion (hopefully it'll fix this)! I'll implement it as I have other changes to make from other reviews.